config/spinnaker.yml

```diff
@@ -231,8 +231,8 @@ services:
 
   stackdriver:
     enabled: ${SPINNAKER_STACKDRIVER_ENABLED:false}
-    projectName: ${SPINNAKER_STACKDRIVER_PROJECT_NAME:${providers.google.primaryCredentials.project}}
-    credentialsPath: ${SPINNAKER_STACKDRIVER_CREDENTIALS_PATH:${providers.google.primaryCredentials.jsonPath}}   
+    projectName: ${providers.google.primaryCredentials.project}
+    credentialsPath: ${providers.google.primaryCredentials.jsonPath} 
 
```

Comment:
The effect I was after was to have stackdriver write into the deployed spinnaker project, not the project spinnaker was managing.
The proposal would work, but would prefer finding a different way so taht when on GCE it would default to the "other" project presuming they are differe.t